src/OpenFeature.Contrib.Providers.Flagd/Resolver/InProcess/JsonEvaluator.cs

```diff
@@ -61,9 +61,11 @@ internal JsonEvaluator(string selector)
             _selector = selector;
 
             var stringEvaluator = new StringEvaluator();
+            var semVerEvaluator = new SemVerEvaluator();
 
             EvaluateOperators.Default.AddOperator("starts_with", stringEvaluator.StartsWith);
             EvaluateOperators.Default.AddOperator("ends_with", stringEvaluator.EndsWith);
+            EvaluateOperators.Default.AddOperator("sem_ver", semVerEvaluator.Evaluate);
```

Comment:
Also need to register `FractionalEvaluator`

Reply:
Oh I think this is a good point cc @bacherfl

Reply:
good catch, thank you :) will add that